client/prisma/migrations/20220802122130_add_transaction_detail/migration.sql

```diff
@@ -0,0 +1,50 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `txHashes` on the `lockups` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[last_seen_block]` on the table `listener` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[lockupId,user]` on the table `lockups` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[proposal_id]` on the table `proposals` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[address]` on the table `voters` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- DropIndex
+DROP INDEX "listener_last_seen_block_key";
+
+-- DropIndex
+DROP INDEX "lockups_lockupId_user_key";
+
+-- DropIndex
+DROP INDEX "proposals_proposal_id_key";
+
+-- DropIndex
+DROP INDEX "voters_address_key";
+
+-- AlterTable
+ALTER TABLE "lockups" DROP COLUMN "txHashes";
+
+-- CreateTable
+CREATE TABLE "Transaction" (
```

Comment:
This migration file is overriden by the next one right? If you want I think you can clean it up by force deleting some already applied migrations in `_prisma_migrations` psql table in case you would want to get rid of this file

Reply:
Good call, will delete as we don't need a deep history in dev.